### ISSUE TITLE
AN 2.1.4 installed but nothing happen blender 2.8

### ISSUE THREAD

**Issue by jumpyjumpo** [JacquesLucke/animation_nodes#1098]
Hello,

I'm new to AN so I follow every steps to install it properly on Blender 2.8.

I downloaded the absolut last version of blender 2.8 this morning, installed AN. It seems to install perfectly with no message error and I can see the Animation Node outliner and start to add node tree, but when I try some absolut basics stuffs (like just put an object instancer) nothing happen in viewport and i can't found no duplication of my object in my scene, and I can't figure it out.

I tried the latest version of AN from here, and the Chris Penninger build one, but I have the exact same issue.

Already try to install reinstall Blender and AN.

AN works perfectly when I tried this on Blender 2.79.

What's happen?

Thanks a lot !

**Comment by OmarSquircleArt** [comment]
> Animation Node outliner

Do you have a red border over your node tree? Are you sure your execution options are in order? Do you have something printed on the terminal/console?

**Comment by jumpyjumpo** [comment]
No red border.

My execution options are as follow: 

Auto execution ON
always OFF
tree changed ON
frame changed ON
Property changed ON

I manage to make a object instancer work by clicking on Execute node tree.

But a simple position transformation by clicking the same Execute node tree give me a "not able to setup the execution units" message. See the picture below.

I'm sorry if this is not a bug, i'm really new to it 
![Capture](https://user-images.githubusercontent.com/50735405/57922126-4be5df00-789f-11e9-92e1-85b4e05fbd3e.PNG)
!

**Comment by OmarSquircleArt** [comment]
Well, that's a red border :smile: . I want you to open the console by pressing *Toggle Console* above the message you see. Then copy the message you see there and post it here.

**Comment by jumpyjumpo** [comment]
haha okay ! ^^'

```
Read prefs: C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\config\userpref.blend
found bundled python: C:\Users\Thibaut\Desktop\blender-2.80-2a9214eaaf04-win64\blender-2.80.0-git.2a9214eaaf04-windows64\2.80\python
Registered Animation Nodes
Traceback (most recent call last):
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\utils\handlers.py", line 44, in always
    handler()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\events.py", line 36, in sceneUpdated
    evaluateRaisedEvents()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\events.py", line 44, in evaluateRaisedEvents
    event_handler.update(event.getActives())
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\utils\recursion.py", line 12, in wrapper
    result = function(*args, **kwargs)
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\event_handler.py", line 24, in update
    afterExecution()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\execution\auto_execution.py", line 19, in afterExecution
    scene.update()
AttributeError: 'Scene' object has no attribute 'update'






Traceback (most recent call last):
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\execution\units.py", line 58, in setupExecutionUnits
    unit.setup()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\execution\main_execution_unit.py", line 26, in setup
    exec(self.setupCodeObject, self.executionData, self.executionData)
  File "setup: 'NodeTree'", line 10, in <module>
KeyError: 'bpy_prop_collection[key]: key "Object Instancer" not found'
Traceback (most recent call last):
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\operators\execute_tree.py", line 21, in execute
    nodeTree.execute()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\base_types\node_tree.py", line 92, in execute
    self._execute()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\base_types\node_tree.py", line 105, in _execute
    success = unit.execute()
  File "C:\Users\Thibaut\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\animation_nodes\execution\main_execution_unit.py", line 73, in raiseNotSetupException
    raise ExecutionUnitNotSetup()
animation_nodes.problems.ExecutionUnitNotSetup

location: <unknown location>:-1

location: <unknown location>:-1
```

here is the console content

**Comment by 3DSinghVFX** [comment]
Hi, @OmarSquircleArt, I change this **scene.update** to **scene.update_tag()** in file "auto_execution.py" and it has solved the problem.

**Comment by 3DSinghVFX** [comment]
But now there is another problem of the dependency graph (depsgraph):

![Screenshot from 2019-05-18 02-08-30](https://user-images.githubusercontent.com/30294746/57954929-fd255d00-7911-11e9-84a8-3ae934f4a679.png)

**Comment by OmarSquircleArt** [comment]
Thanks! Looks like this is caused by the recent API change introduced in [rBe693918d4074](https://developer.blender.org/rBe693918d40741b0839cea84d2e0c386cc262e0c3). We will need to update some more things as well in order for things to work again. I don't have much time, but I will try to update that as soon as possible.

**Comment by OmarSquircleArt** [comment]
Fixed in 0872685.
